### PR TITLE
Bring updates from Master to Preview

### PR DESCRIPTION
**Pull request by bikamani**
To do:

Get confirmation from service team for API versions to use on preview.

### CODE REVIEW

File: device/iot-device-samples/pnp-device-sample/temperature-controller-device-sample/pom.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <dependencies>
         <dependency>
             <groupId>com.microsoft.azure.sdk.iot.provisioning</groupId>
-            <artifactId>provisioning-device-client</artifactId>
+            <artifactId>provisioning-device-client-preview</artifactId>
```
Comment (bikamani) [inline device/iot-device-samples/pnp-device-sample/temperature-controller-device-sample/pom.xml:16]: Note: auto merge brought in wrong artifact, manually changed it for pnp samples

**Review by timtay-microsoft — APPROVED** [review]

**Review by vinagesh — APPROVED** [review]

**Pull request merged**